src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_sun calendar--month-length_29">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
це зайве, тобі достатньо nth-child використовувати, окремо кожен клас прописувати не треба

Reply:
Внесла изменения.